README.md

```diff
@@ -1,7 +1,73 @@
-# OpenFeature specification
+# OpenFeature specification (draft)
 
 [![Roadmap](https://img.shields.io/static/v1?label=Roadmap&message=public&color=green)](https://github.com/orgs/openfeatureflags/projects/1)
 [![Contributing](https://img.shields.io/static/v1?label=Contributing&message=guide&color=blue)](https://github.com/openfeatureflags/.github/blob/main/CONTRIBUTING.md)
 [![Code of Conduct](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](https://github.com/openfeatureflags/.github/blob/main/CODE_OF_CONDUCT.md)
 
-> :warning: Details are coming soon!
+This repository includes the top level architecture for the project,
+and also specifications for all key interfaces and components of the project.
+
+> :warning: This is a draft version that includes key concepts.
+> More Details are coming soon!
+> See [this issue](https://github.com/openfeatureflags/spec/issues/4) for the scope of the alpha version.
+
+## Key concepts
+
+The OpenFeature project follows the following architecture concepts:
+
+- Cloud Native architecture, based on the common technology stacks
+  and open source projects from the [Cloud Native Landscape](https://landscape.cncf.io/). 
+- Extensibility for open source and commercial offerings.    
+- Unified API and SDK for feature flag management in different technology stacks.
+- Feature Flag evaluation logic is externalized into a separate container or a service.
+  Introducing new rules and evaluation logic should not require updating all the apps.
+- Feature flag configurations are stored along with the project and
+  hence managed along with the application (a.k.a "Feature Flags as Code")
+- Built-in support for observability tools tools and standards
+  like OpenTelemetry and possibly CloudEvents
+
+## Architecture
+
+The architecture includes a control plane and also containerized Feature Flag services.
+The system is managed through Custom Resource Definitions (CRDs) or APIs,
+both are served by the control plane.
+
+* **Feature Flag Resource** is a custom CRD type that defines feature flags and evaluation logic for the system.
+* **Control Plane** includes the Feature Flag controller services, `etcd` based storage,
+  and also API services that handle external requests and CRDs.
+* **Feature Flag Service (_FF Service_)** provides feature flag management and evaluation to user applications.
```

Comment:
Not clear if this is the "Feature Flag controller services" referenced in control plane or a separate service with a similar name?

Reply:
It is listed as a separate entity here and in the paragraph above. Also there is a diagram below that shows 2 entities as separate ones